src/main/java/emis.command/ToDoCommand.java

```diff
@@ -0,0 +1,22 @@
+package main.java.emis.command;
+import main.java.emis.*;
+import main.java.emis.task.ToDo;
+
+public class ToDoCommand extends Command {
+    private String description;
+
+    public ToDoCommand(String description) {
+        this.description = description;
+    }
+
+    @Override
+    public void execute(TaskList t, Ui ui, Storage s) {
+        t.addTask(new ToDo(this.description));
+        s.updateStorage();
+    }
+
+    @Override
+    public boolean isExit() {
```

Comment:
I wonder if theres a better way of implementing this? As the number of commands grow, every command will have an isExit(), but yet only one command will ever be isExit() = true?